.jenkins/sarus_run_test.sh

```diff
@@ -17,18 +18,18 @@ gcloud auth configure-docker
 pip install -r requirements.txt
 
 # Run c12 regression test on each Docker image
-declare -a tags=("hpc" "hpc-serialize")
+module load sarus
+declare -a tags=("$tagname" "${tagname}-serialize")
```

Comment:
```suggestion
tags="$tagname ${tagname}-serialize"
```

Reply:
Would rather not change the working code